server.js

```diff
@@ -1,5 +1,6 @@
 const express = require("express");
 const cors = require("cors");
+const bodyParser = require("body-parser");
```

Comment:
Nowadays you can just use `express.json()` (turns out the `express` package now includes `bodyParser` by default, and so `express.json()` is the same thing as `bodyParser.json` in middleware - this was news to me :sweat_smile: )